R/fetch-token.R

```diff
@@ -11,7 +11,7 @@ fetch_token <- function(scopes, ...) {
     # TODO(craigcitro): Expose error handling and/or silencing here.
     try({
       token <- f(scopes, ...)
-    })
+    }, silent = TRUE)
     if (!is.null(token)) {
```

Comment:
@craigcitro I think this is neutral to positive for the time being. Or let me know if it's exactly counter to what you had in mind with the TODO. If you expand on that, I could try to implement. I assume it's about messaging (or possibly stopping) as you work your way down the list?